src/jira_timemachine/__init__.py

```diff
@@ -331,7 +335,7 @@ def timemachine(config, days):
             source_worklog.description = worklog_msg.format(source_worklog)
             source_worklog.issue = config_dict.get('issue_map', {}).get(
                 source_worklog.issue, config_dict['destination_jira']['issue'])
-            source_worklog.author = config_dict['destination_jira']['login']
+            source_worklog.author = destination_tempo.account_id
```

Comment:
what's the account_id format? can we get that informaton from jira if tempo doesn't provide it?

Reply:
These look like a number followed by an UUID or just a hex string; we get them from JIRA.

This line is about posting worklogs by the user we are using on the destination JIRA: we transform a source JIRA worklog object into a destination JIRA worklog object.